hbase-common/src/main/java/org/apache/hadoop/hbase/ChoreService.java

```diff
@@ -151,20 +149,18 @@ public synchronized boolean scheduleChore(ScheduledChore chore) {
 
     try {
       if (chore.getPeriod() <= 0) {
-        LOG.info("Chore {} is disabled because its period is not positive.", chore);
+        LOG.info("Chore {} is disabled because its period is not positive", chore);
         return false;
       }
-      LOG.info("Chore {} is enabled.", chore);
+      LOG.info("Chore {} is enabled", chore);
       chore.setChoreServicer(this);
       ScheduledFuture<?> future =
           scheduler.scheduleAtFixedRate(chore, chore.getInitialDelay(), chore.getPeriod(),
             chore.getTimeUnit());
       scheduledChores.put(chore, future);
       return true;
     } catch (Exception exception) {
-      if (LOG.isInfoEnabled()) {
-        LOG.info("Could not successfully schedule chore: " + chore.getName());
-      }
+      LOG.info("Could not successfully schedule chore: " + chore.getName());
```

Comment:
Why did you remove the check?

Reply:
@HorizonNet The `LOG.info` method will check internally if the log level is set to Info before emitting a log message, no need to do it twice.  Since the log message is INFO level, it's almost certainly going to be enabled when running in production so it's not worth being too clever with regards to performance so I wouldn't normally bother making it parameterized if it's not already.  However, I will make that adjustment since I'll emit the `chore` object instead of its `getName()`.

Also, this should be an ERROR level logging, to include the stack trace.

Reply:
that's a good catch. This exception handling used to also be used as a non-error condition way of disabling the chore. agree that now it should probably call more attention to itself.